potassco/clingo.h

```diff
@@ -209,6 +211,8 @@ class AbstractStatistics {
 	virtual const char* key(Key_t mapK, size_t i) const = 0;
 	//! Returns the element stored in the map under the given name.
 	virtual Key_t       get(Key_t mapK, const char* at) const = 0;
+	//! Create a new Element of type t in the map.
+	virtual Key_t add(Key_t arr, const char* at, Statistics_t t) = 0;
```

Comment:
The Key_t parameter should be named mapK.
Furthermore, overloading `add` in this way might not be the best idea because
````
stats.add(someKey, 0, someType);
```` 
is ambiguous (int could be promoted to size_t or converted to const char*). So, one would always have to write `stats.add(someKey, size_t(0), someType);`